spec/game_logic_spec.rb

```diff
@@ -0,0 +1,39 @@
+require '../lib/game_logic.rb'
+
+describe Board do
+  describe '#update_board' do
+    it 'updates the items in the board' do
+      b = Board.new
+      b.update_board('X', 6)
+      expect(b.board). to eql([1, 2, 3, 4, 5, 'X', 7, 8, 9])
+    end
+  end
+
+  describe '#winner_pattern' do
+    it 'returns the winner pattern states' do
+      b = Board.new
+      expect(b.winner_pattern). to eql([[1, 2, 3], [7, 8, 9], [4, 5, 6], [1, 4, 7],
+                                        [2, 5, 8], [3, 6, 9], [1, 5, 9], [3, 5, 7]])
+    end
+  end
```

Comment:
:bulb: You can also check this method after making a few moves to see if it's reflecting the correct layout. Since this is a hardcoded part of the logic, it can fail easily. You may want to cover it with a proper test case.